docs/pages/versions/unversioned/sdk/gesture-handler.md

```diff
@@ -14,7 +14,7 @@ An API for handling complex gestures. From the project's README:
 
 ## Installation
 
-<InstallSection packageName="react-native-gesture-handler" href="https://docs.swmansion.com/react-native-gesture-handler/" />
+<InstallSection packageName="react-native-gesture-handler" href="https://docs.swmansion.com/react-native-gesture-handler/docs/#installation" />
```

Comment:
```suggestion
<InstallSection packageName="react-native-gesture-handler" href="https://docs.swmansion.com/react-native-gesture-handler/docs/" />
```

i think it's clear enough where the installation is on the page and i'm afraid of depending on an anchor as those tend to change